scapy/contrib/rtcp.py

```diff
@@ -105,7 +137,18 @@ class RTCP(Packet):
         # HEADER
         BitField('version', 2, 2),
         BitField('padding', 0, 1),
-        BitFieldLenField('count', 0, 5, count_of='report_blocks'),
+        ConditionalField(
+            BitFieldLenField('count', 0, 5),
+            lambda pkt: pkt.packet_type not in (205, 206)
```

Comment:
You can't have a conditional test ABOVE the value it's using inside packet dissection :/

Reply:
That's a bummer.
I'll have to come up with something better than just a bunch of `ConditionalField`s then, there is too many of them anyway for my likings

Reply:
Are you sure that you're reading the RFC right? It doesn't really make sense as it just can't be dissected without heuristics